media/com_users/js/admin-users-groups.js

```diff
@@ -1,32 +1,34 @@
+/**
+* PLEASE DO NOT MODIFY THIS FILE. WORK ON THE ES6 VERSION.
+* OTHERWISE YOUR CHANGES WILL BE REPLACED ON THE NEXT BUILD.
+**/
+
 /**
  * @copyright   Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
  * @license     GNU General Public License version 2 or later; see LICENSE.txt
  */
 
 Joomla = window.Joomla || {};
 
-(function(Joomla) {
-	'use strict';
```

Comment:
Please review at this point

Reply:
I remove Joomla parameter is this correct ?

Reply:
Technically its not wrong to remove the parameter, but it has benefits to keep it: 
* uglify can minify it (smaller filesize)
* its clear what dependencies this script needs
* its easier to test

so i recommend (re-)adding all dependencies and pass them as params